src/style.css

```diff
@@ -1,3 +1,73 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+
+  --active-link-color: #00acdc;
+  --header-height: 60px;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  display: inline-block;
+  line-height: 0;
+}
+
+.list {
+  display: flex;
+  list-style-type: none;
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`. In this case, you should remove the `list-style-type: none;` from the `.list` class and instead create a specific class that can be applied to `ul` elements that should not have list styles.